app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/lightning-talk.mdx

```diff
@@ -0,0 +1,47 @@
+---
+meta:
+  title: Lightning Talk
+  description: Short tech-related presentation from members to members.
+hero:
+  Hero: UndrawConference
+tags: [memberresources]
+order: 7
+---
+
+import LeadText from '~/components/content/LeadText';
+import TextContainer from '~/components/content/TextContainer';
+
+<TextContainer background="light" showBackToTopLink={false}>
+
+<LeadText>
+
+Occasionally, we hold Lightning Talk sessions. Virtual Coffee's Lightning Talk is a great way to try your hand at public speaking in a supportive and safe place!
+
+</LeadText>
+
+</TextContainer>
+
+## Table of Contents
+
+<TextContainer>
+
+---
+
+## How It Works
+
+In Lightning Talk, a group of members takes turns presenting their talk in a 5-20 minute time range. Our Maintainers will share a Call for Proposals in Virtual Coffee Slack about one month before the event. And we always provide mentorship for those who want it. So we got your back!
```

Comment:
During our Lightning Talks event, a group of members takes turn presenting their 5-20 minute talk. Our Maintainers will share a Call for Proposals in the Virtual Coffee Slack about one month before the event. Nervous about submitting a talk? We've got your back! We always provide mentorship for those who want it.